From: Ole Streicher <olebole@debian.org>
Date: Tue, 16 Jun 2020 12:39:13 +0200
Subject: Use the packaged MathJax

---
 doc/source/conf.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/doc/source/conf.py b/doc/source/conf.py
index 6107a71..ab9d94d 100644
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -238,3 +238,5 @@ plot_html_show_source_link = False
 # Intersphinx to get Numpy and other targets
 intersphinx_mapping = {
     'numpy': ('https://numpy.org/devdocs', None)}
+
+mathjax_path = 'file:///usr/share/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML'
